Tools/TestWebKitAPI/Tests/WebCore/CSSParser.cpp

```diff
@@ -171,7 +171,7 @@ static unsigned computeNumberOfTracks(const CSSValueContainingVector& valueList)
             continue;
         if (is<CSSGridIntegerRepeatValue>(value)) {
             auto& repeatValue = downcast<CSSGridIntegerRepeatValue>(value);
-            numberOfTracks += repeatValue.repetitions() * computeNumberOfTracks(repeatValue);
+            numberOfTracks += repeatValue.repetitions().resolveAsIntegerNoConversionDataRequired() * computeNumberOfTracks(repeatValue);
```

Comment:
What guarantees no conversion data is required?

Reply:
This is an API test, and all the test cases don't require conversion data. If one is added that does, this will assert.